ansible/roles/cassandra-cql-update/files/cassandra.cql

```diff
@@ -560,4 +560,11 @@ insert into sunbird.system_settings (id,field,value) values ('emailUnique','emai
 ALTER TABLE sunbird.geo_location ADD userCountTTL text;
 // adding new column in client-info to save the channel
 ALTER TABLE sunbird.client_info ADD channel text;
-CREATE INDEX inx_ci_clientchannel ON sunbird.client_info(channel);
\ No newline at end of file
+CREATE INDEX inx_ci_clientchannel ON sunbird.client_info(channel);
+
+// Sep 9th 2018 System Initialisation
```

Comment:
System initialisation script shouldn't expect these settings to be present in DB before running the script. e.g. if 'systemInit' is not available in DB then it should be regarded as if the value is 'false' (i.e. system is not yet initialised).

Also the number of variables used by script are too many. This needs to be optimised.

Reply:
Review Comment Incorporated in code